src/sagemaker/clarify.py

```diff
@@ -81,6 +113,32 @@ def __init__(
                 f"Invalid dataset_type '{dataset_type}'."
                 f" Please check the API documentation for the supported dataset types."
             )
+        # parameters for analysis on datasets without facets are only supported for CSV datasets
+        if dataset_type != "text/csv":
+            if predicted_label:
+                raise ValueError(
+                    f"The parameter 'predicted_label' is not supported"
+                    f" for dataset_type '{dataset_type}'."
+                    f" Please check the API documentation for the supported dataset types."
+                )
+            if excluded_columns:
```

Comment:
Same for `excluded_columns. I think they are supported for all data types.

Reply:
Not according to our analysis config logic. link: https://tiny.amazon.com/nqpmexig/codelink

Reply:
Resolving this conversation because we discussed it offline and my changes reflect the current logic.